Sliding-Images-Easy.md

```diff
@@ -0,0 +1,249 @@
+# Sliding Images Easy
+
+This tutorial will walk you through building an image slider using the [jQuery](https://jquery.com/) library.
+
+![GIF showing Slider in action](https://i.gyazo.com/03dc8fa8a0cd34e6f0dc190d852f23c4.gif)
```

Comment:
@atjonathan Add a VERY LARGE (**but optimized small size**) image showing the slides in a browser interface and **linked to the demo codepen**. :stuck_out_tongue_winking_eye: :wink: